docs/Configuration Parameters.md

```diff
@@ -49,7 +49,8 @@ The below tables represent the different sections of the configuration file.
 | token                | /etc/opt/csl/csl-node/long-lived-token | path to token      |
 | certFile             |     /etc/ssl/certs/csl-ca-cert.pem     | path the cert file |
 
-### SOTA
+Note: ubuntuAptSource tag is no longer used in/After 4.2.0 Version 
```

Comment:
@tsirlapu this is unclear--which version is the first one that no longer uses the tag?

Reply:
Fixed

Reply:
> Fixed

Text still reads "Note: ubuntuAptSource tag is no longer used in/After 4.2.0 Version". I would suggest just "Note: ubuntuAptSource tag has been removed"

Reply:
(also, please change the other instances of "Note: ubuntuAptSource tag is no longer used in/After 4.2.0 Version")

Reply:
@gblewis1 "Note: ubuntuAptSource tag has been removed" will not mention from which version it has been removed, is that okay?